apps/web/src/data/contract/requests/getBids.ts

```diff
@@ -34,9 +38,13 @@ const readAuctionBidEvents = async (auctionAddress: string): Promise<Bid[]> => {
   })
 }
 
-async function getBids(auction: string, tokenId: string | number): Promise<Bid[]> {
+async function getBids(
+  chain: Chain,
```

Comment:
does it make sense to just explicitly pass chainId around instead of the whole chain object?

Reply:
reworked data layer / hooks to accept CHAIN_ID instead of Chain objects now